src/libraries/System.Diagnostics.Process/ref/System.Diagnostics.Process.cs

```diff
@@ -10,7 +10,6 @@ namespace Microsoft.Win32.SafeHandles
     public sealed partial class SafeProcessHandle : Microsoft.Win32.SafeHandles.SafeHandleZeroOrMinusOneIsInvalid
     {
         public SafeProcessHandle(System.IntPtr existingHandle, bool ownsHandle) : base (default(bool)) { }
-        public override bool IsInvalid { get { throw null; } }
```

Comment:
Unrelated to annotation work. Override missing from implementation but present in ref source.